app/screens/thread/thread.tsx

```diff
@@ -124,6 +127,11 @@ const Thread = ({
                             rootPost={rootPost!}
                         />
                     </View>
+                    <ScheduledPostIndicator
+                        isCRTEnabled={isCRTEnabled}
+                        rootId={rootId}
+                        channelId={EMPTY_STRING}
```

Comment:
this makes me thing that channelId in `SchedulePostIndicator` should be optional

Reply:
Done.